data/json/recipes/other/tool.json

```diff
@@ -177,5 +177,26 @@
     "book_learn": [ [ "mag_survival", 2 ], [ "textbook_survival", 1 ], [ "cookbook", 2 ] ],
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "can_food_unsealed", 2 ], [ "scrap", 4 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "razor_blade",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "skills_required": [ "fabrication", 3 ],
+    "difficulty": 3,
+    "time": 8000,
```

Comment:
If it's forged, would need much more time to craft it.

Reply:
Again, not forged.